src/action.js

```diff
@@ -49,16 +50,16 @@ module.exports = async function run({
     const pr = pull_request || (await client.getPullRequest(PR_NUMBER))
 
     const isDependabotPR = pr.user.login === dependabotAuthor
-    if (!isDependabotPR) {
+    if (!isDependabotPR && !SKIP_VERIFICATION) {
       return logWarning('Not a dependabot PR, skipping.')
     }
 
     const commits = await client.getPullRequestCommits(pr.number)
-    if (!commits.every(commit => commit.author?.login === dependabotAuthor)) {
+    if (!commits.every(commit => commit.author?.login === dependabotAuthor) && !SKIP_VERIFICATION) {
```

Comment:
```suggestion
    if (!SKIP_VERIFICATION && !commits.every(commit => commit.author?.login === dependabotAuthor)) {
```

Place variables in the front for fast return.